src/components/Generic/BaseAccordion.js

```diff
@@ -21,16 +21,20 @@ import MarkedIcon from '../Icons/MarkedIcon';
 import CollectionIcon from '../Icons/CollectionIcon';
 
 const AccordionHeader = ({
-  item, expanded, fromDetailsPanel, activeCollectionId,
+  item, expanded, fromDetailsPanel, activeCollectionId, fromDateAccordion,
 }) => {
   const { headerLabel, headerCount, resourceIds } = item;
+  const headerStyle = [styles.header];
+  if (fromDetailsPanel && fromDateAccordion) {
+    headerStyle.push({ paddingLeft: 30 });
```

Comment:
This works, without using `StyleSheet.create`?  

(eg: `styles.header` references the result of a call to `StyleSheet.create({...)}`)